client/src/Utils/rfFunctions.ts

```diff
@@ -0,0 +1,20 @@
+export function calculateDate(start: Date, count: number, sampleRate: number) {
+  let time = start.getTime() + count / sampleRate;
+  return new Date(time).toISOString();
+}
+
+export function calculateFrequency(freq: number) {
+  if (freq.toString().length >= 10) {
+    return freq / 1000000000 + ' GHz';
```

Comment:
`
```suggestion
    return freq / 1_000_000_000 + ' GHz';
```

makes easier to read numberrs if you add _ for lage numbers

Reply:
or 1e9 if the number is already a float